src/templates/verilogeditor.js

```diff
@@ -111,11 +111,46 @@ class Verilog_editor {
     return str;
   }
 
+  set_Signals_Tb(space, m) {
+    var str = '';
+    for (let x = 0; x < m.length; ++x) {
+      if (m[x]['type'] === '') {
+        if (m[x]['direction'] === "input") {
+          str += space + 'reg ' + m[x]['name'] + ' = 0;\n';
+        } else {
+          str += space + 'wire ' + m[x]['name'] + ';\n';
+        }
+
+      }
+      else {
+        const regex = /\[(.*?)\]/;
+        let type = m[x]['type'].match(regex);
+        if (type === null) {
+          type = '';
+        } else {
+          type = type[0];
+        }
+        if (m[x]['direction'] === "input") {
+          if (type === '') {
+            str += space + 'reg ' + type + ' ' + m[x]['name'] + ' = 0;\n';
```

Comment:
Try to use "`" and " `". Example: 

str += \`${space} reg ${type} ${m[x]['name']} = 0;\n;\`

Reply:
Same comment than before. I consider the style refactoring a separate issue

Reply:
That is ok with the old code, but no with the new code added. Anyway, I think that isn't an old or new style problem. In this case In this case I think the code is cleaner with \`